packages/blockchain-sdk-solana/src/messaging/solana-messaging.ts

```
@@ -273,6 +275,18 @@ export class SolanaThread implements Thread {
   markAsRead(): Promise<void> {
     return Promise.resolve(undefined);
   }
+
+  addMembers(command: AddMembersCommand): Promise<void> {
+    return Promise.resolve(undefined);
```

let's throw UnsupportedOperationError here